vm/vm.go

```diff
@@ -613,13 +614,11 @@ func (vm *VM) GetStatelessBlock(ctx context.Context, blkID ids.ID) (*chain.State
 	}
 	vm.verifiedL.RUnlock()
 
-	// not found in memory, fetch from disk if accepted
-	stBlk, err := vm.GetDiskBlock(blkID)
-	if err != nil {
-		return nil, err
+	// not found in memory, check if getting last accepted
```

Comment:
I think this should be returning the genesis block as well

Reply:
nice catch